### ISSUE TITLE
Homebrew Tap is broken

### ISSUE THREAD

**Issue by rsdy** [FiloSottile/yubikey-agent#79]
I get the following response from `brew update`:
```
Error: Fetching /opt/homebrew/Library/Taps/filippo.io/homebrew-yubikey-agent failed!
```

Trying to untap/retap will result in a password prompt for Github.

**Comment by FiloSottile** [comment]
Thank you for the report. I added back a `master` branch with just the Homebrew Formula, and I will replace it with a Tap migration entry once Homebrew/homebrew-core#71440 is merged.

Could you try a `brew update` and check if it's working?

**Comment by cerw** [comment]
```
brew install yubikey-agent                                                                                                                            <update_wp_core ✗>
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 5 taps (shivammathur/php, homebrew/cask-versions, homebrew/core, homebrew/cask and homebrew/services).
==> Updated Formulae
Updated 70 formulae.
==> New Casks
kiwi-for-gmail
==> Updated Casks
Updated 51 casks.

==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "yubikey-agent".

```

**Comment by rsdy** [comment]
@FiloSottile I still get a password prompt when I `brew tap filippo.io/yubikey-agent`:
```
% brew tap filippo.io/yubikey-agent

==> Tapping filippo.io/yubikey-agent
Cloning into '/opt/homebrew/Library/Taps/filippo.io/homebrew-yubikey-agent'...
Username for 'https://github.com': ^C
```

**Comment by FiloSottile** [comment]
New `brew tap` invocations are broken and will not get fixed, once Homebrew/homebrew-core#71440 is merged `brew install yubikey-agent` will just work without the custom tap. What was fixed and verified by another reporter is `brew update` for existing taps.
